seed/prices.json

```diff
@@ -1,757 +1,772 @@
 [
   {
     "address": "0x3472A5A71965499acd81997a54BBA8D852C6E53d",
-    "price": 11.11,
-    "updatedAt": 1644856645671
+    "price": 7.81,
```

Comment:
this is a routine pricing update to accommodate quicker cold starts wrt pricing tests